client/classic/upe.js

```diff
@@ -3,127 +3,46 @@
 /**
  * Internal dependencies
  */
-// import './style.scss';
-// import {
-// 	PAYMENT_METHOD_NAME_CARD,
-// 	PAYMENT_METHOD_NAME_UPE,
-// } from '../constants.js';
-// import { getConfig } from 'utils/checkout';
-// import WCPayAPI from '../api';
-// import enqueueFraudScripts from 'fraud-scripts';
-// import { getFontRulesFromPage, getAppearance } from '../upe-styles';
+import './style.scss';
+import { getConfig } from '../utils';
+import WCStripeAPI from './api';
+import { getFontRulesFromPage, getAppearance } from '../upe-styles';
+
+const PAYMENT_METHOD_NAME_CARD = 'stripe';
+const PAYMENT_METHOD_NAME_UPE = 'stripe_upe';
 
 jQuery( function ( $ ) {
```

Comment:
We can also `import $ from jQuery` if we prefer to use that syntax here. You can look at `client/api/index.js` for an example of that 🙂

Reply:
I like the `import $` but I'm not sure how to implement the early return when we don't have stripe keys:
```
if ( ! publishableKey ) {
	// If no configuration is present, probably this is not the checkout page.
	return;
}
```

Reply:
Yeah, that's a good point 🤔 
I'm sure it's possible, but it's probably going to be way worse than using what we already have 😅 

I think we could maybe `import jQuery` (or `import jQuery from jQuery`?) to be explicit about the dependency, but not change anything in the code? 🤔

Reply:
Added the explicit import 👍🏼